### PR TITLE
Added test and example for -c/--creator feature

### LINKED ISSUE #131: test, and document `creator` feature
the [-c/--creator](https://jupysql.ploomber.io/en/latest/api/magic-sql.html) feature is not well documented; we only describe what it is, but there are no runnable examples. also, I quickly look at the code and seems that we don't have any unit tests.

[based on this](https://docs.sqlalchemy.org/en/20/core/engines.html#sqlalchemy.create_engine.params.creator), looks like creator is passed to sqlalchemy to initialize a connection. this feature sounds very similar to the one we added recently for connecting to [existing engines](https://jupysql.ploomber.io/en/latest/howto.html#connect-to-existing-engine). I think we should keep both to remain 100% compatible with ipython-sql  but recommend users switch to the new feature.

### PR DESCRIPTION
**Pull request by DaveOkpare**
## Describe your changes
Added test and example for -c/--creator feature

## Issue number

Closes #131 

## Checklist before requesting a review

- [x] Performed a self-review of my code
- [x] Formatted my code with [`pkgmt format`](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#linting-formatting)
- [x] Added [tests](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#testing) (when necessary).
- [x] Added [docstring](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#documenting-changes-and-new-features) documentation and update the [changelog](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#changelog) (when needed)

----
:books: Documentation preview :books:: https://jupysql--456.org.readthedocs.build/en/456/

### CODE REVIEW

File: src/tests/test_magic.py [hunk 1]
```diff
@@ -519,6 +519,18 @@ def test_close_connection_with_alias(ip, tmp_empty):
     assert "two" not in Connection.connections
 
 
+def test_creator_no_argument_raises(ip):
+    result = ip.run_cell("%sql --creator")
+    assert isinstance(result.error_in_exec, UsageError)
+
+
+def test_creator(ip):
```
[the remark this answers is not in the document]
Comment (DaveOkpare) [reply src/tests/test_magic.py:527]: Alright, I'm on it!

[the remark this answers is not in the document]
Comment (tonykploomber) [reply src/tests/test_magic.py:527]: Probably need to rebase

**Comment by DaveOkpare** [comment]
> @DaveOkpare any updates on this one?

Apologies, certain databases require a docker image setup. I'm struggling with setting up integration tests. 

PostgreSQL and SQLite work without an image setup.

Screen Shot 2023-05-31 at 10 57 20 AM

**Comment by edublancas** [comment]
@DaveOkpare: no worries. just add unit tests for sqlite and duckdb

**Pull request closed without merging**